src/main/kotlin/com/frc2036/comp/GameController.kt

```diff
@@ -12,6 +12,9 @@ import kotlin.random.Random
 @RequestMapping(value = ["/api"])
 class GameController {
     private val tournament = Tournament(1,"observe0","admin0", true)
+    init {
+        tournament.updateActiveGames()
+    }
```

Comment:
Currently, the games aren't assigned to boards until a player calls `/api/move_needed`. Probably good to init them when the server starts